12/umbraco-vendr/core/how-to-guides/migrating-from-vendr-to-umbraco-commerce.md

```diff
@@ -0,0 +1,319 @@
+WHERE type LIKE '%Vendr.%';
+```
+
+6. Update template paths:
+
+```sql
+UPDATE umbracoCommerceEmailTemplate
+SET templateView = REPLACE(templateView, '/App_Plugins/Vendr/templates/email', '/Views/UmbracoCommerce/Templates/Email')
+WHERE templateView LIKE '%/Vendr/%';
+UPDATE umbracoCommercePrintTemplate
+SET templateView = REPLACE(templateView, '/App_Plugins/Vendr/templates/print', '/Views/UmbracoCommerce/Templates/Print')
+WHERE templateView LIKE '%/Vendr/%';
+UPDATE umbracoCommerceExportTemplate
+SET templateView = REPLACE(templateView, '/App_Plugins/Vendr/templates/email', '/Views/UmbracoCommerce/Templates/Export')
+WHERE templateView LIKE '%/Vendr/%';
+```
+
+7. Update the migrations log:
+
+```sql
+UPDATE umbracoCommerceMigrations
+SET migration = REPLACE(migration, 'Vendr.', 'Umbraco.Commerce.')
+WHERE migration LIKE 'Vendr.%';
+```
+
+8. Update the activity logs:
+
+```sql
+UPDATE umbracoCommerceActivityLog
+SET eventType = REPLACE(eventType, 'vendr/', 'commerce/')
+WHERE eventType LIKE 'vendr/%';
+```
+
+### Step 4: Finalizing the migration
+
+1. Delete any obj/bin folders in your projects to ensure a clean build.
+2. Recompile all projects and ensure all dependencies are restored correctly
+3. Change license files - TBD
+4. Update any payment gateways that uses a global webhook:
+
+```none
+https://{site_url}/umbraco/commerce/payment/callback/{payment_provider_alias}/{payment_method_id}/
+```
+
+5. Run the project.
+
+Before moving on to migrating the pakcages and payment providers, it is recommended to ensure everything works as expected.
+
+## Migrate Umbraco Commerce Checkout
+
+Throught the following steps we will migrate the Checkout package from Vendr to Umbraco Commerce.
+
+### Step 1: Backup
+
+1. Make a backup of any custom templates and Umbraco Commerce UI configuration files.
+2. Make a note of all configuration values on the Vendr.Checkout checkout node.
+
+### Step 2: Uinstall Vendr.Checkout
+
+1. Delete Vendr.Checkout generated checkout nodes.
+2. Delete all Vendr.Checkout generated Document Types.
+3. Delete all Vendr.Checkout generated Data Types.
+4. Uninstall the Vendr.Checkout nuget package:
+
+```bash
+dotnet remove package Vendr.Checkout
+```
+
+5. Delete any remaining `~/App_Plugins/VendrCheckout` files and folders.
+
+### Step 3: Install Umbraco.Commerce.Checkout
+
+1. Install the Umbraco.Commerce.Checkout package
+
+    ```bash
+    dotnet add package Umbraco.Commerce.Checkout
+    ```
+
+2. In the settings section, locate the Umbraco Commerce Checkout dashboard and reinstall into previous location.
+
+### Step 4: Reapply custom changes
+
+1. Copy any custom configuration files back into the solution. 
+2. Copy any custom Views in to the `~/Views/UmbracoCommerceCheckout/` folder.
+
+## Migrate Payment Providers
+
+Through the following steps we will migrate payment providers used for Vendr into Umbraco Commerce.
+
+### Step 1: Replace dependencies
+
+1. Remove any installed Vendr packages
```

Comment:
Hasn't this already been completed in an earlier step?

Reply:
As per my comment further up, I've made a mistake here in my last comment. This isn't about replacing installed payment providers, this section is actually about custom payment providers and how to upgrade those, so when it's saying remove any installed Vendr packages, it's referring to payment provider removing any Vendr nuget dependencies it has.

Reply:
I've updated the heading to "Migrate Custom Payment Providers"